Update README.md (#421)
aboutyou/dart_packages: packages/sign_in_with_apple/sign_in_with_apple/README.md

Before you can start integrating (or even testing) Sign in with Apple you need a [paid membership to the Apple Developer Program](https://developer.apple.com/programs/). Sign in with Apple is one of the restricted services which is not available for free with just an Apple ID ([source](https://developer.apple.com/programs/whats-included/)).

#### Apple Mail Relay

> Since the users are able to use the private email relay, it is necessary to add a SPF record to the domains used to send emails, please read more here : https://developer.apple.com/help/account/configure-app-capabilities/configure-private-email-relay-service/

Reviewer: ```suggestion
Since the users can use the private email relay, it is necessary to add an SPF record to the domains used to send emails, please read more here: https://developer.apple.com/help/account/configure-app-capabilities/configure-private-email-relay-service/
```

> Without it your service will not be able to send emails to the user that choose to use Apple's private relay, and the email will not be delivered.

Reviewer: ```suggestion
Without it, your service will not be able to send emails to users who choose to use Apple's private relay, and the emails will not be delivered.
```

> 

Reviewer: ```suggestion
```
